app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -488,6 +493,33 @@ function AcuantCapture(
     }
   }
 
+  function onSelfieCaptureSuccess({ image }: { image: string }) {
+    // TODO analytics
+    // TODO are there possible errors here?
```

Comment:
Would we want to keep these TODOs (and line 519) on merge, or add them as technical notes to a future ticket?